dashboard/config/initializers/devise.rb

```diff
@@ -328,7 +328,15 @@
 
   require 'cookie_helpers'
   Warden::Manager.after_set_user do |user, auth|
-    auth.cookies[environment_specific_cookie_name("_user_type")] = {value: user.teacher? ? "teacher" : "student", domain: :all, httponly: true}
+    user_type =
+      if user.teacher?
+        "teacher"
+      elsif user.under_13?
+        "student_y"
```

Comment:
`student_y` doesn't seem like the most descriptive name - is it short for `young_student`?

Reply:
`young_student` is what we use in UI tests to indicate anyone younger than 13, so it seems we could be consistent at least with that.